pkg/middleware/dashboard_redirect.go

```diff
@@ -62,23 +59,21 @@ func RedirectFromLegacyPanelEditURL(cfg *setting.Cfg) func(c *models.ReqContext)
 
 func RedirectFromLegacyDashboardSoloURL(cfg *setting.Cfg) func(c *models.ReqContext) {
 	return func(c *models.ReqContext) {
-		slug := c.Params("slug")
 		renderRequest := c.QueryBool("render")
 
-		if slug != "" {
-			url, err := getDashboardURLBySlug(c.OrgId, slug)
-			if err != nil {
-				return
-			}
-
-			if renderRequest && strings.Contains(url, cfg.AppSubURL) {
-				url = strings.Replace(url, cfg.AppSubURL, "", 1)
-			}
-
-			url = strings.Replace(url, "/d/", "/d-solo/", 1)
-			url = fmt.Sprintf("%s?%s", url, c.Req.URL.RawQuery)
-			c.Redirect(url, 301)
+		url, err := getDashboardURLBySlug(c.OrgId)
```

Comment:
Think `RedirectFromLegacyDashboardSoloURL` function should be removed since no longer used?